app/controllers/admin/uphold_reports_controller.rb

```diff
@@ -0,0 +1,30 @@
+require 'csv'
+
+module Admin
+  class UpholdStatusReportsController < AdminController
+    def index
+      @uphold_status_reports = UpholdStatusReport.
+        group('(EXTRACT(YEAR FROM created_at))::integer').
+        group('(EXTRACT(MONTH FROM created_at))::integer').
+        order('2 DESC, 3 DESC').count
```

Comment:
```suggestion
        order('2 DESC, 3 DESC')
```

Reply:
Why this change? Since we're doing a `group` I want get a count of the amount in each report